src/app.js

```diff
@@ -133,6 +133,11 @@ const wizardOptions = {
 
 router.use(wizard(steps, fields, wizardOptions));
 
+router.use((req, res, next) => {
+  res.locals.htmlLang = req.lng;
```

Comment:
What is this for? The functionality is working in F2F but I can't see this code there 🤔